### PR TITLE
QNAP product attribute is incorrect

### PR DESCRIPTION
**Pull request by a573net**
Hello,

I found an incorrect QNAP entry in multipath.conf.
The product attribute was "iSCSI_Storage", but the actual QNAP product attribute was "iSCSI Storage". Underscores should be spaces.

I suggested writing 2 blocks of underscore and space in the forum below.
Please check the forum below for details.
https://xcp-ng.org/forum/topic/5194/multipath-conf-is-incorrect

I was suggested by forum users to give feedback to the fork source.

I use multipathing with iSCSI in QNAP.
Because of the different multipath speeds, I had to set a fixed priority.
I noticed this mistake with the multipath -ll command and posted.

### CODE REVIEW

**Comment by MarkSymsCtx** [comment]
Two things

- You need to sign off the commit for DCO, do `git commit --amend -s` and then force push to your branch
- This should replace the incorrect block of configuration instead of creating a new one

**Review by MarkSymsCtx — CHANGES_REQUESTED** [review]
Two things

You need to sign off the commit for DCO, do git commit --amend -s and then force push to your branch
This should replace the incorrect block of configuration instead of creating a new one

**Comment by olivierlambert** [comment]
Do we know if QNAP changed their product attribute or the `iSCSI_Storage` was wrong since the start?

**Comment by MarkSymsCtx** [comment]
> Do we know if QNAP changed their product attribute or the `iSCSI_Storage` was wrong since the start?

I suspect wrong from the start

**Comment by olivierlambert** [comment]
So in that case indeed, it's best to modify it instead of adding it (if it's wrong from the start, it means there's no chances to break something existing/used)

**Comment by a573net** [comment]
I am not used to operating github.
I have operated several times.
I'm sorry.

**Review by MarkSymsCtx — CHANGES_REQUESTED** [review]
That's still wrong, you still have two copies of the QNAP configuration.

**Comment by MarkSymsCtx** [comment]
And you need to clean up the commit history, read up on `git rebase` - https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

**Comment by a573net** [comment]
Everyone, please help me.
I'm not sure. I can't rebase well.
I think you should delete # 576 itself.
Please tell me how to do it.

**Comment by a573net** [comment]
If you can close this # 576 with the authority of the master, please close it.

**Comment by a573net** [comment]
I'm sorry for sending multiple messages.
I use the web interface.
Can I delete # 576 with "**Close pull request**" under this comment field?

**Comment by MarkSymsCtx** [comment]
I've done the rebasing required here but you still need to sign-off the commit. Do the following from where have the code checked out. (assuming that your git origin is your fork, if not change origin to the name of your fork's remote)
```
git checkout patch-2
git reset --hard origin/patch-2
git commit -s --amend
git push origin patch-2 -f
```

**Review by MarkSymsCtx — APPROVED** [review]

**Comment by a573net** [comment]
Thank you very much for helping me.
I executed the command you told me.
Please check status.

**Review by MarkSymsCtx — APPROVED** [review]

**Review by edwintorok — APPROVED** [review]
The commit message has some duplication (copy/paste or squashing issue?), but the change itself looks fine.

**Comment by MarkSymsCtx** [comment]
> The commit message has some duplication (copy/paste or squashing issue?), but the change itself looks fine.

a573net, if you want to do another `git commit --amend` and update the comment to remove the duplication then that's fine otherwise it's fine as it is with the message duplicated.

**Comment by a573net** [comment]
> a573net, if you want to do another `git commit --amend` and update the comment to remove the duplication then that's fine otherwise it's fine as it is with the message duplicated.

I am not familiar with git, so I am not confident in the operation. The message column is dirty, but I want to finish it as it is. If you have to change it, tell me the git command.
Thank you for helping me.

**Comment by stormi** [comment]
You'd do:
```
git commit --amend
```
Edit the message in the editor that opens.

Then:
```
git push --force-with-lease
```

**Comment by a573net** [comment]
Thank you for helping me.
I corrected the message. Please check.

**Review by rosslagerwall — APPROVED** [review]

**Comment by MarkSymsCtx** [comment]
Sorry, to be a pain but as this goes into the history of the code can you make one more small change to the commit message and change the first line to a brief summary of the change? Something like
```
Update product identification for QNAP iSCSI storage
```
And then make the rest a bit less like an email, removing the initial `Hello, ` will be sufficient for that.

**Comment by a573net** [comment]
Thank you for following me.
I summarized the comments.
Please check.

**Comment by a573net** [comment]
Oops.
I accidentally deleted the signoff line for DCO.

**Review by MarkSymsCtx — APPROVED** [review]

**Review by rosslagerwall — APPROVED** [review]

**Comment by a573net** [comment]
Do I need to operate something? Should I wait without doing anything?

**Comment by olivierlambert** [comment]
IMHO, I think now it's just waiting for 2 other reviews, so it looks that you don't need to do anything else :)

**Review by TimSmithCtx — APPROVED** [review]

**Comment by olivierlambert** [comment]
Congrats and thank you @a573net for your contribution :partying_face:

**Comment by a573net** [comment]
Thank you for your cooperation!

**Pull request merged**
